app.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "ROSE",
+  "description": "ROSE project car race game.",
+  "repository": "https://github.com/RedHat-Israel/ROSE",
+  "keywords": ["race", "game", "python"]
```

Comment:
This should also describe:
- The server is started by running the ```rose-server``` script
- Port 8880 should be open to receive web clients
- Port 8888 should be open to receive rose-client(s)

See the updated issue for more details.

Reply:
Indeed, tried the button now from your fork https://github.com/mayukh18/ROSE/ => https://cben-rose-test.herokuapp.com, says "Application Error", heroku logs show:
```
2017-10-21T17:31:29.442324+00:00 heroku[router]: at=error code=H14 desc="No web processes running" method=GET path="/" host=cben-rose-test.herokuapp.com request_id=68b48dde-386d-4756-a4bd-a5901ea49fe5 fwd="141.226.169.57" dyno= connect= service= status=503 bytes= protocol=https
```
https://cben-rose-test.herokuapp.com:8880/ doesn't work either, connection refused.

- I don't see anything in https://devcenter.heroku.com/articles/app-json-schema or elsewhere about specifying what port we listen on; we [have to respect PORT env var](https://devcenter.heroku.com/articles/dynos#local-environment-variables) that heroku provides.

- On the outside, heroku router only listens on http[s] :80 and :443.  That's fine for user interface, maybe will need to document that :8880 is not needed with heroku.

- The nastier implication is you [can't have 2nd outside port for rose-client](https://stackoverflow.com/questions/40029675/open-two-different-ports-in-heroku-app).  The client will have to connect on 80/443, and will need another mechanism for server to differentiate connection type (URL path? content-type?  user-agent (meh)?  custom header?).

Reply:
Yes I too didn't find any documentation related to servers and ports in https://devcenter.heroku.com/articles/app-json-schema and related articles. Second thing which I know is true is that heroku pre-assigns $PORT. What we need is a **Procfile**